src/app/clusters/barrier-control-server/barrier-control-server.c

```diff
@@ -41,8 +41,14 @@
 #include "barrier-control-server.h"
 #include "af.h"
 
+#include <assert.h>
+
+#ifndef MILLISECOND_TICKS_PER_DECISECOND
+#define MILLISECOND_TICKS_PER_DECISECOND 1000
```

Comment:
I would think this should be `100`.

But even better would be to just put:
```
#define MILLISECOND_TICKS_PER_DECISECOND (MILLISECOND_TICKS_PER_SECOND / 10)
```
in a header that gets included from all the things we care about (e.g. `types_stub.h` for the moment?).  And probably put the definition of `MILLISECOND_TICKS_PER_SECOND` in the same header.

That said, as a followup we should investigate all the uses of this define and see whether they make sense in a world in which non-HAL code doesn't know about ticks at all.   Please file a followup issue on that?